src/FlipMove.js

```diff
@@ -394,6 +369,32 @@ class FlipMove extends Component<ConvertedProps, FlipMoveState> {
     });
   }
 
+  // eslint-disable-next-line camelcase
+  UNSAFE_componentWillReceiveProps(nextProps: ConvertedProps) {
```

Comment:
I believe that prefix UNSAFE_ is not good option for long time. Instead we should something for long life.